src/main/resources/application.yaml

```diff
@@ -59,3 +61,23 @@ white-label:
     title: Zeebe Simple Monitor
     css.path: css/custom.css
     js.path: js/custom.js
+---
+spring:
+  config:
+    activate:
+      on-profile: kafka
+  datasource:
+    url: ${POSTGRES_DATASOURCE_URL:jdbc:postgresql://localhost:5432/postgres}
+    username: ${POSTGRES_DATABASE_USER:postgres}
+    password: ${POSTGRES_DATABASE_PASSWORD:temp}
+    driverClassName: org.postgresql.Driver
+  jpa:
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+    hibernate:
+      ddl-auto: update
+
+kafkaBootstrapServers: ${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+kafkaSchemaRegistryUrl: ${SCHEMA_REGISTRY_URL:http://localhost:8081}
+kafka:
+  streams:
+    maxMessageSizeBytes: ${KAFKA_MAX_MESSAGE_SIZE_BYTES:5242880}
```

Comment:
when using this on the cluster, you need to have auth setup (e.g. a `kafka-user` and secrets), this would be done inside the zeebe-simple-monitor-chart repo. Ask your local integrations person for help with that

Reply:
We will update kafka and deployment related configuration in another PR